stable/kong/values.yaml

```diff
@@ -142,3 +143,42 @@ postgresql:
   postgresDatabase: kong
   persistence:
     enabled: false
+
+# Ingress Controller:
+# A controller watches for kubernetes resources defined with a
+# CustomResourceDefinition that represent the configuration of a deployment.
```

Comment:
Kong Ingress Controller's primary purpose is to satisfy Ingress resources created in k8s. 
It uses CRDs for more fine grained control over routing and for Kong specific configuration.

@merqurio Could you update the description to communicate this? Thanks!

Reply:
Done. Thanks,